src/Common/src/Interop/Windows/Ole32/Interop.CoGetObjectContext.cs

```diff
@@ -0,0 +1,16 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Runtime.InteropServices.ComTypes;
+
+internal static partial class Interop
+{
+    internal static partial class Ole32
+    {
+        [DllImport(Libraries.Ole32, PreserveSig = false)]
+        internal static extern IStream CreateStreamOnHGlobal(IntPtr hGlobal, int fDeleteOnRelease);
```

Comment:
change the `int` to `bool`. The default marshalling of `bool` is a Win32 `BOOL` which makes the types more accurate.